src/nanoemoji/write_variable_font.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Assembles UFOs into a variable font.
+
+Based on https://gist.github.com/anthrotype/c11515065a4aa6549f9d5f2dfdcf8f23"""
+
+from absl import app
+from fontTools import designspaceLib
+from nanoemoji import config
+import ufo2ft
+import ufoLib2
+
+def only(filter_fn, iterable):
+    it = filter(filter_fn, iterable)
+    result = next(it)
+    assert next(it, None) is None
+    return result
+
+
+def main(argv):
+    ufos = tuple(filter(lambda a: a.endswith(".ufo"), argv[1:]))
```

Comment:
i'm seeing a lot of `filter()` lately, is that a new functional programming fad? :) 
doing `tuple(filter(lambda ..., iterable))` is not necessarily faster or more readable than using a list/tuple comprehension or a generator expressions, e.g. `tuple(v for v in iterable if ...)`. It's more a style thing I guess. I don't mind anyway, just want to point this out.

Reply:
Tbh I think whether I write filter vs a comprehension varies based on what I was working on prior to writing it. If I stop and think about it I'd say I prefer the comprehension/generator style for trivial examples like this one.

Reply:
...I now want to go change a bunch of recently added filters to use the non-filter style.

Reply:
[I did]